generator/integration_tests/generator_integration_test.cc

```diff
@@ -97,9 +97,7 @@ class GeneratorIntegrationTest : public testing::TestWithParam<std::string> {
                           "Codegen C++ Generator");
 
     product_path_ = "generator/integration_tests/golden/";
-    // The golden files were last generated on 2021, update next time you
-    // re-generate said files.
-    copyright_year_ = "2021";
+    copyright_year_ = CurrentCopyrightYear();
```

Comment:
I would rather you did 2022, that way we will not have the same breaks on 2023-01-01 we had in 2022-01-01.

Frankly, I would rather we pinned these to a specific year, but since I don't know how to do that ...

Reply:
Done.